dashboard/app/models/user.rb

```diff
@@ -462,7 +459,7 @@ def email_matches_for_oauth_upgrade
   validates_presence_of :email_preference_form_kind, if: -> {email_preference_opt_in.present?}
 
   # Validations for adding parent email notifications
-  before_validation :parent_email_preference_setup, if: :parent_email_preference_opt_in_required?
+  before_validation :parent_email_preference_setup, if: -> {parent_email_preference_email.present? && user_type == 'student'}
```

Comment:
One issue this is going to cause, which I don't know if we care about, is that now students will be able to add a parent_email during sign up (sorry I don't think I wrote a unit test for this edge case).

1. Student is on 2nd page of sign up process
2. Student checks "I am a Parent..."
3. Student adds parent_email field
4. Student unchecks "I am a Parent..."
5. Student fills out the rest of the required fields and submits
6. Now the student has parent_email defined (although no email_preference).